### PR TITLE
Pin to updated vcloud-core

### PR DESCRIPTION
**Pull request by alexmuller**
I'm not sure if this is a correct approach or whether incrementing the major version is a good idea.

### CODE REVIEW

File: vcloud-tools.gemspec [hunk 1]
```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 1.9.3'
 
-  s.add_runtime_dependency 'vcloud-core'
+  s.add_runtime_dependency 'vcloud-core', '>= 1.1.0'
```
Comment (rjw1) [inline vcloud-tools.gemspec]: are those the right magic symbols. i can never remember what ~ does :)

**Comment by joshmyers** [comment]
Looks :+1: to me!

**Comment by mattbostock** [comment]
What's the reason for the major version bump? Does the change break backwards compatibility?

**Comment by ajlanghorn** [comment]
Waiting on responses from @alexmuller before reviewing further.

**Comment by mattbostock** [comment]
_bump_

**Comment by alexmuller** [comment]
I don't think this is the right thing to do anymore. This repository shouldn't be specifying versions like this.

**Pull request closed without merging**